collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2954,6 +2980,18 @@ public void onNumberPickerValueSelected(int widgetId, int value) {
     }
 
     /**
+     * getter for currentView variable. This method should always be used
+     * to access currentView as an ODKView object to avoid inconsistency
+     **/
+    @Nullable
+    private ODKView getCurrentViewIfODKView() {
+        if (currentView != null && currentView instanceof ODKView) {
```

Comment:
Strictly speaking, the `null` check is not necessary -- https://stackoverflow.com/a/2950415/137744

Reply:
Thanks for bringing this up. Wasn't aware of that